.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2
+  browser-tools: circleci/browser-tools@1.0.0
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.5.1-stretch-node
+        environment:
+          RAILS_ENV: test
+          DATABASE_URL: "postgresql://root@localhost/adventure-time_test?pool=5"
+          BUNDLER_PATH: vendor/bundle
+          BUNDLER_JOBS: 4
+          YARN_CACHE_FOLDER: ~/.cache/yarn
+      - image: circleci/postgres:9.3-alpine-ram
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: adventure-time_test
+          TZ: "/usr/share/zoneinfo/America/New_York"
+
+    executor: ruby/default
+    steps:
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
+      - checkout
+      - run:
+          name: Which bundler?
+          command: bundle -v
+      - run:
+            bundle update capistrano
```

Comment:
why this update?

Reply:
Was getting this error 

```
Downloading capistrano-2.15.9 revealed dependencies not in the API or the
lockfile (net-scp (>= 1.0.0)).
Either installing with `--full-index` or running `bundle update capistrano`
should fix the problem.
```

Just doing `--full-index` didn't completely fix it, but updating did. Trying to find another way to avoid this